src/screens/ImportWalletScreen.tsx

```diff
@@ -218,34 +356,42 @@ export const ImportWalletScreen = (props: Props) => {
     // 7. check if its private key (segwit address P2SH) TODO
     // 7. check if its private key (legacy address) TODO
   };
-
-  return (
-    <ScreenTemplate
-      footer={
-        <>
-          <Button disabled={isButtonDisabled} title={i18n.wallets.importWallet.import} onPress={onImportButtonPress} />
-          <FlatButton
-            containerStyle={styles.scanQRCodeButtonContainer}
-            title={i18n.wallets.importWallet.scanQrCode}
-            onPress={onScanQrCodeButtonPress}
+  render() {
+    const { validationError, text } = this.state;
+    return (
+      <ScreenTemplate
+        footer={
+          <>
+            <Button
+              disabled={!text || !!validationError}
```

Comment:
label is also needed here probably, `!text || !!validationError || !label`

Reply:
If label is not provided wallet will be named as other imported wallet with its type as name